lib/compress/zstd_compress.c

```diff
@@ -2547,6 +2549,7 @@ static void ZSTD_copyBlockSequences(ZSTD_CCtx* zc)
     outSeqs[i].matchLength = outSeqs[i].offset = outSeqs[i].rep = 0;
     seqStoreSeqSize++;
 
+    ZSTD_memcpy(zc->blockState.nextCBlock->rep, updatedRepcodes.rep, sizeof(repcodes_t));
```

Comment:
We shouldn't need this line. `nextCBlock->rep` is set by the block compressor already.

An assert that they are the same would be nice though.